eox_nelp/processors/xapi/event_transformers/initialized_events.py

```diff
@@ -0,0 +1,47 @@
+"""
+Transformers for initialized events.
+
+Classes:
+    InitializedCourseTransformer: Transformer for the event nelc.eox_nelp.initialized.course
+"""
+
+from tincan import Activity, ActivityDefinition, LanguageMap, Verb
+
+from eox_nelp.edxapp_wrapper.event_routing_backends import XApiTransformer, XApiTransformersRegistry, constants
+from eox_nelp.processors.xapi import constants as eox_nelp_constants
+from eox_nelp.utils import get_course_from_id
+
+
+@XApiTransformersRegistry.register("nelc.eox_nelp.initialized.course")
```

Comment:
Which was the criteria to select this name?Mainly prefix nelc

Reply:
https://github.com/eduNEXT/eox-nelp/blob/master/eox_nelp/signals/receivers.py#L52

Reply:
Ok yes, maybe in the receiver I would ask if there is a convention to give that names xD

Reply:
too late, but yes there are some comments  https://github.com/openedx/event-tracking/blob/master/eventtracking/tracker.py#L6